.github/workflows/python-spark3.0-tests.yml

```diff
@@ -0,0 +1,34 @@
+name: Python Tests for Spark 3.0
```

Comment:
I think that we can merge the two actions into one just by adding the spark version into the matrix, please look at:
https://github.com/neo4j-contrib/neo4j-spark-connector/pull/301/files#diff-b803fcb7f17ed9235f1e5cb1fcd2f5d3b2838429d4368ae4c57ce4436577f03fR15

Reply:
and then you can manage what you need to do inside the python script